src/ckms.rs

```diff
@@ -10,25 +10,48 @@
 //! variant of this algorithm is fundamentally flawed, an issue which the
 //! authors correct in their "Space- and Time-Efficient Deterministic Algorithms
 //! for Biased Quantiles over Data Streams"
-
 use std;
 use std::cmp;
 use std::fmt::Debug;
 use std::ops::{Add, AddAssign, Div, Sub};
 
-#[derive(Debug, Clone, PartialEq)]
+#[derive(Debug, Clone)]
 #[cfg_attr(feature = "serde_support", derive(Serialize, Deserialize))]
-struct Entry<T: Copy> {
+struct Entry<T>
+where
+    T: Copy + PartialEq,
+{
     v: T,
     g: usize,
     delta: usize,
 }
 
```

Comment:
Can we get a comment as to why we implement PartialEq / ParitalOrd ourselves?

Reply:
Resolved.